### ISSUE TITLE
wlroots >=0.17 required

### ISSUE THREAD

**Issue by EmnichtdaYT** [WillPower3309/swayfx#305]
### Please read the following before submitting:
- Proprietary graphics drivers, including nvidia, are not supported. Please use the open source equivalents, such as nouveau, if you would like to use Sway.
- Please do NOT submit issues for information from the github wiki. The github wiki is community maintained and therefore may contain outdated information, scripts that don't work or obsolete workarounds.
  If you fix a script or find outdated information, don't hesitate to adjust the wiki page.

### Please fill out the following:
- **Swayfx Version:**
  - `sway -v`

- **Debug Log:**
  - Run `sway -d 2> ~/sway.log` from a TTY and upload it to a pastebin, such as gist.github.com.
  - This will record information about sway's activity. Please try to keep the reproduction as brief as possible and exit sway.
  - Attach the **full** file, do not truncate it.

- **Configuration File:**
  - Please try to produce with the default configuration.
  - If you cannot reproduce with the default configuration, please try to find the minimal configuration to reproduce.
  - Upload the config to a pastebin such as gist.github.com.

- **Stack Trace:**
  - This is only needed if sway crashes.
  - If you use systemd, you should be able to open the coredump of the most recent crash with gdb with
    `coredumpctl gdb sway` and then `bt full` to obtain the stack trace.
  - If the lines mentioning sway or wlroots have `??` for the location, your binaries were built without debug symbols. Please compile both sway and wlroots from source and try to reproduce.

- **Description:**
  - The steps you took in plain English to reproduce the problem.

**Comment by WillPower3309** [comment]
should be fixed by https://github.com/wlrfx/aur-packages/commit/e9ad03c1d2c1dc588c41ed20d2eccaf3b5c32e73, please let me know if this addressses it :)

**Comment by EmnichtdaYT** [comment]
In the PKGBUILD, `PKG_CONFIG_PATH` is set to `'/usr/lib/wlroots0.16/pkgconfig'` is this intentional? Shouldn't it be /usr/lib/pkgconfig?

**Comment by millionsofplayers** [comment]
having this issue aswell, i assume replying here gets me notifications through email for this thread?

i also heard i probably did a partial upgrade because of this error, am i safe installing stuff on my system or should i just wait

**Comment by WillPower3309** [comment]
should be fixed by https://github.com/wlrfx/packages/commit/f6e33d122dc406205617275aa1da85a467d30d74. Please reopen if this is not the case
